tests/raftstore_cases/test_merge.rs

```diff
@@ -357,13 +357,25 @@ fn test_node_merge_brain_split() {
     cluster.must_transfer_leader(1, new_peer(1, 1));
     let pd_client = Arc::clone(&cluster.pd_client);
     let region = pd_client.get_region(b"k1").unwrap();
+
     cluster.must_split(&region, b"k2");
+    cluster.must_put(b"k11", b"v11");
+    cluster.must_put(b"k21", b"v21");
+    for i in 1..4 {
```

Comment:
Can you leave a comment why we do it 4 times?

Reply:
I don't think it's necessary, `must_get_equal` loops itself.

Reply:
Just make sure all stores have replicated latest update.